src/kubernetes-tool/templates/app.vue

```diff
@@ -20,25 +20,41 @@ limitations under the License.
             <SplashScreen @result="resultSet" />
         </div>
         <div v-else>
-            <Header :title="i18n.templates.app.title">
-                <template v-slot:description>
-                </template>
-                <template v-slot:header>
-                </template>
-                <template v-slot:buttons>
-                    <a class="button" @click="mainMenu">{{ i18n.templates.shared.mainMenu }}</a>
-                </template>
-            </Header>
+            <div class="columns">
+                <div class="column is-2" style="height: 100%; top: 0; position: sticky;">
+                    <div v-for="allCats in [Categorisation.getAll()]">
+                        <aside class="menu" style="margin-left: 20px; margin-top: 20px;">
+                            <span v-for="key in allCats.keys()" :key="key.name">
+                                <p class="menu-label">
+                                    {{ key.name }}
+                                </p>
+                                <ul class="menu-list">
+                                    <li v-for="item in allCats.get(key)" :key="item.fp" style="font-size: 15px">
+                                        <a @click="navbarHook(item.fp)">
+                                            {{ item.fp }}
+                                        </a>
+                                    </li>
+                                </ul>
+                            </span>
+                        </aside>
+                    </div>
+                </div>
+                <div class="column">
+                    <Header :title="i18n.templates.app.title">
+                        <template v-slot:description>
+                        </template>
+                        <template v-slot:header>
+                        </template>
+                        <template v-slot:buttons>
+                            <a class="button" @click="mainMenu">{{ i18n.templates.shared.mainMenu }}</a>
+                            <a class="button" @click="collapseHook">{{ i18n.templates.app.collapseAll }}</a>
```

Comment:
This button currently just toggles the state of each "collapisble" - this should store it's own state and actually collpase/uncollapse everything. The text can then also reflect this.